### PR TITLE
Update TotalConnect for config flow

### PR DESCRIPTION
**Pull request by austinmroczek**
## Proposed change

Adds discussion of how to add TotalConnect via the user interface.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/32126

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Shuffle a little bit in the file, put the configuration piece on top. This is more in line with how we like it.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by austinmroczek** [comment]
I adjusted this PR after the review was complete because the code PR ended up having a breaking change, and I had to account for that here.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Here and there I see some extra whitespaces, can you remove them? 😄

**Comment by klaasnicolaas** [comment]
I actually meant between the sentences, what you have now adjusted was not necessary.

**Comment by austinmroczek** [comment]
Per discussion with balloob we un-did the breaking change in the code.  Configuration.yaml entry is still supported, so I added it back in here.

File: source/_integrations/totalconnect.markdown [hunk 1]
```diff
@@ -6,46 +6,25 @@ ha_category:
...
-To enable TotalConnect, add the following lines to your `configuration.yaml`:
+To enable TotalConnect via the user interface, go to **Configuration** > **Integrations** > the "plus" button > Total Connect.
 
+To enable TotalConnect via `configuration.yaml` add the following lines:
 ```yaml
 totalconnect:
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
+    username: !secret your_username
+    password: !secret your_password
```
Comment (frenck) [inline source/_integrations/totalconnect.markdown]: Please use placeholders:

```suggestion
    username: YOUR_USERNAME
    password: YOUR_PASSWORD
```

File: source/_integrations/totalconnect.markdown [hunk 2]
```diff
@@ -6,46 +6,25 @@ ha_category:
...
-
-The TotalConnect API has limited zone type information.  Home Assistant device class `door` is assigned to TotalConnect door, window, perimeter, motion sensor, and most alarm panel buttons.  The sensor will appear as `True` if the door is open (either fault or triggered in TotalConnect) and `False` otherwise. Device class `smoke` is assigned to TotalConnect smoke detectors and buttons with physical alarm panel "Response Type" setting of "Fire No Verification".  The sensor will appear as `True` if smoke is detected.  Device class `gas` is assigned to TotalConnect carbon monoxide detectors.  The sensor will appear as `True` if gas is detected.
-
 ## Configuration
 
-To enable TotalConnect, add the following lines to your `configuration.yaml`:
+To enable TotalConnect via the user interface, go to **Configuration** > **Integrations** > the "plus" button > Total Connect.
 
+To enable TotalConnect via `configuration.yaml` add the following lines:
```
Comment (frenck) [inline source/_integrations/totalconnect.markdown:20]: Newline before starting a fenced code block:

```suggestion
To enable TotalConnect via `configuration.yaml` add the following lines:

```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, @austinmroczek 

2 Small suggestions left. Could you take a look? Thanks! 👍

**Review by frenck — APPROVED** [review]
Thanks for the updates @austinmroczek! 👍

**Pull request merged**